v3.0/tidb-in-kubernetes/deploy/gcp-gke.md

```diff
@@ -46,25 +46,22 @@ If the GCP project is new, make sure the relevant APIs are enabled:
 {{< copyable "shell-regular" >}}
 
 ```bash
-gcloud services enable cloudresourcemanager.googleapis.com && \
-gcloud services enable cloudbilling.googleapis.com && \
-gcloud services enable iam.googleapis.com && \
-gcloud services enable compute.googleapis.com && \
-gcloud services enable container.googleapis.com
+gcloud services enable cloudresourcemanager.googleapis.com \
+    cloudbilling.googleapis.com cloud services enable iam.googleapis.com \
+    compute.googleapis.com container.googleapis.com
 ```
 
 ### Configure Terraform
 
-The terraform script expects three environment variables. You can let Terraform prompt you for them, or `export` them in the `~/.bash_profile` file ahead of time. The required environment variables are:
+The terraform script expects three variables to be provided by the user. You can let Terraform prompt you for them, or define them in a `.tfvars` file of your choice. The three variables are:
 
-* `TF_VAR_GCP_CREDENTIALS_PATH`: Path to a valid GCP credentials file.
-    - It is recommended to create a new service account to be used by Terraform. See [this page](https://cloud.google.com/iam/docs/creating-managing-service-accounts) to create a service account and grant `Project Editor` role to it.
-    - See [this page](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) to create service account keys, and choose `JSON` key type during creation. The downloaded `JSON` file that contains the private key is the credentials file you need.
-* `TF_VAR_GCP_REGION`: The region to create the resources in, for example: `us-west1`.
-* `TF_VAR_GCP_CREDENTIALS_PATH`: Path to a valid GCP credentials file.
-    - It is recommended to create a new service account to be used by Terraform as shown in the below example.
+* `GCP_CREDENTIALS_PATH`: Path to a valid GCP credentials file.
+    - It is recommended to have a separate service account to be used by Terraform. See [this page](https://cloud.google.com/iam/docs/creating-managing-service-accounts) for more information. The `Project Editor` role will grant sufficient permissions for provisioning the necessary infrastructure resources.
```

Comment:
It is already implied that Project Editor is sufficient permissions to do anything.

```suggestion
    - It is recommended to have a separate service account to be used by Terraform. See [this page](https://cloud.google.com/iam/docs/creating-managing-service-accounts) for more information. `./create-service-account.sh` will create such a service account with minimal permissions.
```